Update WebExtension API processId (#23958)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/webnavigation/getallframes/index.md

- `errorOccurred`
  - : `boolean`. True if the last navigation in this frame was interrupted by an error, i.e. the {{WebExtAPIRef("webNavigation.onErrorOccurred", "onErrorOccurred")}} event fired.
- `processId` {{optional_inline}} {{deprecated_inline}}

>   - : `integer`. This value is never set in modern browsers. It used to represent the ID of the process running the renderer for this tab.

Reviewer: Will this be entirely clear to English as a foreign language readers? Consider changing throughout to.
```suggestion
  - : `integer`. This value is not set in modern browsers. Previously, it represented the ID of the process running the renderer for this tab.
```
